source/tests/consistent/descriptor/test_dpa3.py

```
@@ -156,6 +159,7 @@ def skip_pt(self) -> bool:
             fix_stat_std,
             n_multi_edge_message,
             precision,
+            add_chg_spin_ebd,
```

Ruff (0.15.2)

[warning] Unpacked variable `add_chg_spin_ebd` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)